From bfba6445a778007f40af5cbfbe725e12c0fcafc6 Mon Sep 17 00:00:00 2001
From: Tomas Volf <~@wolfsden.cz>
Date: Tue, 5 Mar 2024 22:25:20 +0100
Subject: [PATCH] gm_utils.cpp: Call clear instead of empty.

Since the intention seem to be to erase the next word, I believe calling empty
was a mistake and it should have been clear.  Empty does nothing in this
context.

* src/gm_utils.cpp (wrap_cstr): Call clear.
---
 src/gm_utils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/gm_utils.cpp
+++ b/src/gm_utils.cpp
@@ -311,7 +311,7 @@ void wrap_cstr(string& wrapped, unsigned
             // trim leading spaces
             std::size_t pos = next_word.find_first_not_of(' ');
             if( pos == std::string::npos )
-                next_word.empty();
+                next_word.clear();
             else if( pos )
                 next_word.erase( 0, pos );
 
